recipes/cereal/meta.yaml

```
@@ -0,0 +1,37 @@
+{% set name = "cereal" %}
+{% set version = "1.2.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/USCiLab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 1921f26d2e1daf9132da3c432e2fd02093ecaedf846e65d7679ddf868c7289c4
+
+build:
+  noarch: generic
+  number: 0
+
+requirements:
+  build:
```

You can also add an empty list normally.
```suggestion
  build: []
```

---

Ah, I see on gitter that this doesn't work ;)